.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+	root: true,
+	env: {
+		node: true,
+		es6: true,
+		browser: true,
+	},
+	parser: 'vue-eslint-parser',
+	parserOptions: {
+		parser: 'babel-eslint',
+		allowImportExportEverywhere: true,
+		ecmaVersion: 2020,
+		sourceType: 'module',
+	},
+	plugins: ['sonarjs'],
+	extends: [
+		'eslint:recommended',
+		'plugin:vue/recommended',
+		'plugin:sonarjs/recommended',
+		'plugin:prettier/recommended',
+		'prettier/vue',
+	],
+	rules: {
+		'no-magic-numbers': [
+			'warn',
+			{ ignoreArrayIndexes: true, ignore: [0, 1, 2, -1, 10] },
```

Comment:
why the 10 though?

Reply:
10 is often used for base10 conversions and is pretty easy to understand in most cases